packages/components/src/components/modal/modal.tsx

```diff
@@ -59,6 +59,8 @@ export class Modal {
   @Prop() closeButtonLabel?: string = 'Close Pop-up';
   /** (optional) title for close button */
   @Prop() closeButtonTitle?: string = 'Close';
+  /** (optional) display close button */
+  @Prop() displayCloseButton?: boolean = true;
```

Comment:
What do you think about making the default `false`, thus changing the prop to something like `hideCloseButton` or `omitCloseButton`? I think "boolean" props should be false by default, I would like to have this by convention…

Reply:
change to `omitCloseButton`